site/gatsby-site/src/components/landing/RandomIncidentsCarousel.js

```diff
@@ -60,7 +62,22 @@ const RandomIncidentsCarousel = () => {
                 {/* The carousel insists on using rounded corners, *
                  * so we have to cover them up on top.            */}
               </div>
-              <Carousel slideInterval={6000} slide={false}>
+              <Carousel
+                slideInterval={6000}
+                slide={false}
+                leftControl={
+                  <FontAwesomeIcon
+                    icon={faArrowCircleLeft}
+                    className="h-8 w-8 text-white bg-gray-800 rounded-full"
```

Comment:
Can you add `shadow` so it sticks out more against white background? And I think the contrast is a bit high. Maybe change it to `bg-gray-500`?